addon/templates/building/edit/entrance/edit/index.hbs

```diff
@@ -17,60 +17,59 @@
       </span>
     </h2>
 
-    <ProjectForm
-      @project={{this.buildingEntrance}}
+    <ModelForm
+      @model={{this.buildingEntrance}}
+      @validations={{this.BuildingEntranceValidations}}
       @onSubmit={{this.saveBuildingEntrance}}
       @translationBase="ember-gwr.buildingEntrance.fields"
-      as |Field|
+      @disabled={{not this.buildingEntrance.street.ESID}}
+      as |Field changeset|
     >
 
       {{#unless this.buildingEntrance.isNew}}
-        <Field @attr="EDID" type="number" readonly disabled />
+        <Field @attr="EDID" @type="number" @disabled={{true}} />
       {{/unless}}
       <Field
         @attr="locality.name.nameLong"
-        required
-        readonly={{not this.buildingEntrance.isNew}}
-        disabled={{not this.buildingEntrance.isNew}}
+        @required={{true}}
+        @readonly={{not this.buildingEntrance.isNew}}
```

Comment:
I don't think there is a `@readonly` parameter (unless you added it in the meantime)

Reply:
No there isn't :sweat_smile: i just added a `@` to every argument.